app/models/host.rb

```diff
@@ -135,7 +135,7 @@ def self.available_os_versions
   def self.arel_inventory_groups(groups, key)
     jsons = groups.map { |group| [{ key => group }].to_json.dump }
 
-    return Arel.sql('1 = 0') if jsons.empty?
+    return AN::InfixOperation.new('=', Arel.sql('1'), Arel.sql('0')) if jsons.empty?
```

Comment:
Can this `1=0` be a (global) constant? 
Just curious, isn't there a way go without `1=0`?

Reply:
In any other case the `scope.none` should be sufficient, but in the arel domain we need the arel version of it. So in every other place we stick to the `none`, but here as the `or` needs to be also in the arel domain, we cannot avoid it. So long story short: this will be the only place to keep it, everywhere else we are using the ActiveRecord way.